plugins/modules/ntnx_pc_unregistration_v2.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Prem Karat
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+module: ntnx_pc_unregistration_v2
+short_description: Unregister a registered remote cluster from the local cluster.
+version_added: 2.1.0
+description:
+    - Unregister a registered remote cluster from the local cluster.
```

Comment:
Please explain this in details. Isnt this used to unregister PC-PC used for availability zones?

Reply:
Explain in terms of PC where you will hit the API.

Reply:
Also mention what it cannot do like PC PE unregister